CRVAnalysis/src/CRVAnalysis.cc

```diff
@@ -109,7 +109,7 @@ namespace mu2e
       MCSummary._nHitCounters=counters.size();
     }
 
-#if 0
+#if 1
```

Comment:
It's Ok if this is a temporary for development purposes.  I would like it gone when you make a production release.

Reply:
The temporary part will be removed soon. It's already out-commented in the #else section.